COM-664 Add new disaggregations for the LTFU report

tmvumbi2 opened the pull request:
Jira ticket: https://jembiprojects.jira.com/browse/COM-664

tmvumbi2 committed:
COM-664 Add new disaggregations for the LTFU report
Adds new age ranges for the LTFU report

lauravignoli commented:
@tmvumbi2 Some feedback regarding the template:

1- Please use Fit all columns on One page for the print settings, otherwise with the current setting (Fit Sheet on One Page) it will always show all patients on one page.

![printSettings](https://user-images.githubusercontent.com/1198827/74651577-c922e500-518c-11ea-8af1-c8d89da1d54c.png)

2- Please center the logo

![logo](https://user-images.githubusercontent.com/1198827/74651392-629dc700-518c-11ea-8494-2a6e5a5c054c.png)

3- Please make sure this column name is visible:

![columnName](https://user-images.githubusercontent.com/1198827/74651465-8a8d2a80-518c-11ea-9109-09d24c208784.png)

4- The template has 12 year dissagregations while the report has 18. Has there been any changes in the spec that I'm not aware of?

lauravignoli commented:
Also, when I run the report with patients, they are not showing.

tmvumbi2 committed:
COM-664 Add some improvements on the template
- update print setup to fit all columns
- center the logo
- center the titles for the 2 tables
- display the number of patients
- zoom in the report so that the column titles are shown correctly

lauravignoli committed:
COM-664 Update column hight on template
